eZ/Publish/API/Repository/UserService.php

```diff
@@ -305,6 +306,20 @@ public function loadUserGroupsOfUser(User $user, $offset = 0, $limit = 25, array
      */
     public function loadUsersOfUserGroup(UserGroup $userGroup, $offset = 0, $limit = 25, array $prioritizedLanguages = []);
 
+    /**
+     * Checks if Content is a user.
+     *
+     *  @since 7.4
+     */
+    public function isUser(Content $content): bool;
+
+    /**
+     * Checks if Content is a user group.
+     *
+     * @since 7.4
```

Comment:
```suggestion
     * @since 7.4
     *
     * @param \eZ\Publish\API\Repository\Values\Content\Content $content
     *
     * @return bool
```